### DISCUSSION TITLE
Screenshot galore: projects built with wxruby3

### DISCUSSION THREAD

**Discussion by rubyFeedback** [mcorino/wxRuby3#335] [Ideas]
It would be nice if the wxruby3 github page (that is, at the main README) could "showcase" some projects built with wxruby3, as (static) images.

Ideally these should be external, but it is also perfectly ok (in my opinion) to feature any applications
you yourself wrote, too, to work around the chicken-egg barrier of adoption.

These applications can be small-ish, of course, but ideally it would also feature a few semi-advanced
examples, featuring some things that can be done with wxruby3. Actually, there are already some examples provided within wxruby3 itself, so these could be used as a starter, such as the excel-like cell / table structure (I forgot which example it was, but I found it yesterday).

Importantly, screenshots should be provided, so that people can easily and visually see what is all
possible via wxruby3. Anyway, this is a very low priority issue request, so I'll close it after suggesting it, to help keep the github tracker somewhat clean.

**Comment by mcorino** [comment]
@rubyFeedback I think this is a better place for these kind of suggestions. This way you can also just keep them open ;-)
